src/clj/game/cards/agendas.clj

```diff
@@ -204,6 +204,19 @@
                  :effect arrange-rd}
         :interactive (req true)}))
 
+   "Better Citizen Program"
+   {:events {:play-event {:req (req (has-subtype? target "Run"))
+                          :delayed-completion true
+                          :msg "give the Runner a tag for playing a run event"
+                          :effect (effect (tag-runner :runner eid 1))
+                          :once :per-turn}
+             :runner-install {:silent (req true)
+                              :req (req (has-subtype? target "Icebreaker"))
+                              :delayed-completion true
+                              :msg "give the Runner a tag for installing an icebreaker"
+                              :effect (effect (tag-runner :runner eid 1))
+                              :once :per-turn}}}
+
```

Comment:
I saw you asking on Slack about DRY'ing this. There's one thing we can do:

```clojure
(let [ability { the delayed-completion, effect, and once key/values go here }]
  {:events {:play-event (assoc ability :req (req (has-subtype ...))
                                       :msg "give the runner...")
            :runner-install (assoc ability :req (...)
                                           :msg ...
                                           :silent ...)})
```

but that arguably isn't much better.

Reply:
Good idea though! - somehow eluded me in my fog yesterday

Reply:
@nealterrell would you like me to give your suggested change a shot?

Reply:
I think it's alright in its current less-DRY version. What you could do is make a function that takes `"Run"` or `"Icebreaker"` and generate the correct ability (with the particulat`req` and `msg`) from that.